FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Fhir/SpecificationProviders/BaseFhirSpecificationProvider.cs

```diff
@@ -30,7 +30,11 @@ public abstract class BaseFhirSpecificationProvider : IFhirSpecificationProvider
         /// </summary>
         private readonly Lazy<FhirCapabilityData> _capabilityData;
 
-        protected static readonly IEnumerable<string> ExcludeTypes = new List<string> { FhirConstants.StructureDefinition };
+        protected static readonly IEnumerable<string> ExcludeTypes = new List<string>
+        {
+            FhirConstants.StructureDefinition,
```

Comment:
Add unit tests to make sure these types are not allowed.

Reply:
Added, thanks.